etc/LATISS/primary_hdu.header

```diff
@@ -38,21 +38,21 @@ RASTART =                      / RA of telescope from AZSTART and ELSTART
 DECSTART=                      / DEC of telescope from AZSTART and ELSTART
 RAEND   =                      / RA of telescope from AZEND and ELEND
 DECEND  =                      / DEC of telescope from AZEND and ELEND
-ROTPA   =                      / Rotation angle relative to the sky (deg)
+ROTPA   =                      / [deg] Rotation angle relative to the sky
 ROTCOORD= 'sky'                / Telescope Rotation Coordinates
-HASTART =                      / [HH:MM:SS] Telescope hour angle at start
+HASTART =                      / Telescope hour angle at start [decimal h]
 ELSTART =                      / [deg] Telescope zenith distance at start
 AZSTART =                      / [deg] Telescope azimuth angle at start
 AMSTART =                      / Airmass at start
-HAEND   =                      / [HH:MM:SS] Telescope hour angle at end
+HAEND   =                      / Telescope hour angle at end [decimal h]
```

Comment:
This is still wrong.

```
HAEND =           / [h]  Telescope hour angle at end
```
is what it should be. (with the right spacing)

Reply:
Fixed to 
```
HAEND =           / [h] Telescope hour angle at end
```